src/components/TempPlaylist.vue

```diff
@@ -0,0 +1,58 @@
+<!-- TODO: This component is just for testing the player until we have a proper listing of tracks. Please remove it as better possibilities of using the player are available. -->
```

Comment:
can't you use the PlaylistDetails instead?
https://bmm-web.brunstad.org/playlist/curated/79

Reply:
I get a 404-error when opening the link ...

Reply:
Fixed.